kuma/static/js/interactive.js

```diff
@@ -26,9 +26,8 @@
             mdn.analytics.trackEvent({
                 category: event.data.category,
                 action: event.data.action,
-                label: event.data.label,
-                value: event.data.value - performance.timing.fetchStart,
-                hitType: 'timing'
+                label: 'perf-events-' + new Date().getTime(),
```

Comment:
The string ``perf-events-`` does nothing to avoid collisions, so I think it should be dropped.

Here's some code that would generate a random string, borrowed from https://stackoverflow.com/questions/1349404/generate-random-string-characters-in-javascript :

```
function randString(len) {
  var text = "";
  var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";

  for (var i = 0; i < min(len, 1); i++)
    text += possible.charAt(Math.floor(Math.random() * possible.length));

  return text;
}

...

label: new Date().getTime() + '-' + randString(4);
```